### PR TITLE
Create fat jar with services CLI so it can be run standalone

### LINKED ISSUE #8540: Generate a "fat jar" for services' CLI tools to run them standalone
Generate a "fat jar" that includes all runtime dependencies so services' CLI tools (in `hedera-node/cli-clients`) can be run standalone (like a normal Java application).

### PR DESCRIPTION
**Pull request by david-bakin-sl**
**Description**:

Create a fat jar for the services cli so it can be run standalone.  Also a script
to run it with (script is a slight modification of `pcli.sh`.

All you need is to copy the fat jar and the script someplace (to the same directory)
and run the script from there.

**Related**:

Fixes #8540 

**Checklist**:

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (manual)

### CODE REVIEW

File: hedera-node/cli-clients/build.gradle.kts [hunk 1]
```diff
@@ -14,16 +14,35 @@
...
+
 plugins {
     id("com.hedera.hashgraph.conventions")
     id("com.hedera.hashgraph.shadow-jar")
 }
 
 description = "Hedera Services Command-Line Clients"
 
+mainModuleInfo {}
+
```
Comment (jjohannes) [inline hedera-node/cli-clients/build.gradle.kts:27]:
```suggestion
```

This does not do anything

File: hedera-node/cli-clients/build.gradle.kts [hunk 2]
```diff
@@ -14,16 +14,35 @@
...
+        archiveClassifier.set("pcli-plugin")
+        manifest {
+            attributes("Main-Class" to "com.swirlds.cli.PlatformCli", "Multi-Release" to "true")
+        }
+    }
+
+tasks.assemble {
+    dependsOn(tasks.shadowJar)
+    dependsOn(cliJar)
+}
```
Comment (jjohannes) [inline hedera-node/cli-clients/build.gradle.kts:48]:
```suggestion
tasks.shadowJar {
    manifest {
        attributes("Main-Class" to "com.swirlds.cli.PlatformCli", "Multi-Release" to "true")
    }
}

tasks.assemble {
    dependsOn(tasks.shadowJar)
}
```

**Review by jjohannes — CHANGES_REQUESTED** [review]
The `shadowJar` task from the shadow plugin, that is already used here, is doing exactly what you need I believe. So I think we should just configure that one with the additional Manifest attributes.

Can you check if what you get after the changes I proposed is sufficient for you?

File: hedera-node/cli-clients/build.gradle.kts (hunk 1 above)
Comment (david-bakin-sl) [reply hedera-node/cli-clients/build.gradle.kts:27]: removed

File: hedera-node/cli-clients/build.gradle.kts (hunk 2 above)
Comment (david-bakin-sl) [reply hedera-node/cli-clients/build.gradle.kts:48]: thank you! works, implemented

**Review by jjohannes — APPROVED** [review]

**Review by JeffreyDallas — APPROVED** [review]

**Review by qnswirlds — APPROVED** [review]

**Review by tinker-michaelj — APPROVED** [review]
LGTM!

**Review by jasperpotts — APPROVED** [review]

**Pull request merged**